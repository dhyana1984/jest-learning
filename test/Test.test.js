import React from 'react';
import { render } from '@testing-library/react-native'
import Test from './Test';

it('renders correctly', () => {
    const prop = (new Date()).toISOString()
    const { toJSON } = render(<Test testProp={prop} />)
    expect(toJSON()).toMatchSnapshot();
});