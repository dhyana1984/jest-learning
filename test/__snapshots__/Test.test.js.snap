// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    this is for test: [object Object]
  </Text>
</View>
`;
